dynamic-demo-plugin/locales/en/plugin__console-demo-plugin.json

```diff
@@ -1,4 +1,7 @@
 {
+  "{{count}} Worker Node": "{{count}} Worker Node",
+  "{{count}} Worker Node_plural": "{{count}} Worker Nodes",
```

Comment:
I think we need to do locales JSON refresh, since the localized key should match the default English value.

Reply:
Not in this case, plural is a special form generated when you use count variable https://www.i18next.com/translation-function/plurals